src/shared/dissect-image.c

```diff
@@ -1475,7 +1478,13 @@ static int dissect_image(
                         /* If we found an embedded signature partition, we are ready, too. */
 
                         m->verity_ready = m->verity_sig_ready = true;
-                        m->partitions[verity->designator == PARTITION_USR ? PARTITION_USR : PARTITION_ROOT].rw = false;
+                        if (verity->designator <= 0) {
```

Comment:
I'd like to make the condition consistent with the above change:
```
if (verity->designator >= 0)
        m->partitions[...].rw = false;
else if (m->partitions[PARTITION_USR_VERITY_SIG].found)
        m->partitions[PARTITION_USR].rw = false;
else if (m->partitions[PARTITION_ROOT_VERITY_SIG].found)
        m->partitions[PARTITION_ROOT].rw = false;
```

Reply:
Done.